arches/app/tasks.py

```diff
@@ -636,12 +636,16 @@ def update_resource_instance_data_based_on_graph_diff(
     logger = logging.getLogger(__name__)
     user = User.objects.get(id=user_id)
 
+    # TODO: update function name to be less specific
+    notify_completion(
+        _("Updating business data based on graph changes."),
```

Comment:
missing positional argument:
```suggestion
        _("Updating business data based on graph changes."),
        user,
```